index.html

```diff
@@ -16,10 +16,33 @@
 
 <body>
 
+    <div class="menu">
+        <div class="container">
+          
+            <div>
+              <div class="nav-toggle" id="navToggle">
+                
+                <img id="navClosed" class="navIcon" src="./Images/Union.png" alt="hamburger menu">
+                <img id="navOpen" class="navIcon hidden" src="./Images/Close.png" alt="close hamburger">
+              </div>
+            </div>
+            
+            <nav>
+              <ul>
+                <li><a href="#portfolio">Portfolio</a></li>
+                <li><a href="#about" >About</a></li>
+                <li><a href="#contact">Contact</a></li>
+               
+              </ul>
+            </nav>
+            
+          </div>
+    </div>
+
     <header class="header">
-        <a href="#">My Logo</a>
+        <a href="#">Gleen</a>
         <nav>
-            <img src="Images/Union.png" alt="hamburger">
+            <img src="./Images/Union.png" alt="hamburger">
```

Comment:
- Great job implementing the mobile menu interactivity 👍🏽, please make sure the menu popup e.g font, colours, images and margins follow the original Figma design.
 Your design | Figma design
![screenshot (3)](https://user-images.githubusercontent.com/44624138/169418764-b4b364e3-c728-4458-8a1c-1419f9732df3.png) | ![screenshot (2)](https://user-images.githubusercontent.com/44624138/169418779-8d93a785-c579-4ee0-9217-8045dd65b228.png)

Reply:
Please make sure the overlay only shows when the hamburger is clicked and disappears when the menu is closed. 👍🏽